kim/pipelines/base.py

```diff
@@ -180,44 +187,44 @@ class StringIntPipeline(Pipeline):
     process_pipes = []
     output_pipes = []
 
-    def __init__(self, mapper_session, field):
-        """Construct a new Pipeline for a :class:`kim.field.Field`.
+    __slots__ = ()
 
-        :param field: An instance of :class:`kim.field.Field` being processed.
-        :param mapper_session: The mapper_session scope this field is being processed in.
-        """
+    def get_pipeline(self, **extra_pipes):
```

Comment:
Not a performance thing but it seems like there is no need to instantiate Pipeline at all now, so this could be a classmethod for clarity?